cmake/patches/protobuf/protobuf_android_log.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/cmake/libprotobuf-lite.cmake b/cmake/libprotobuf-lite.cmake
+index 83e970312..96908991f 100644
+--- a/cmake/libprotobuf-lite.cmake
++++ b/cmake/libprotobuf-lite.cmake
+@@ -102,7 +102,7 @@ if(protobuf_LINK_LIBATOMIC)
+   target_link_libraries(libprotobuf-lite PRIVATE atomic)
+ endif()
+ if(${CMAKE_SYSTEM_NAME} STREQUAL "Android")
+-  target_link_libraries(libprotobuf-lite PRIVATE log)
++  target_link_libraries(libprotobuf-lite PRIVATE -llog)
```

Comment:
Why this change? "-l" is not necessary

Reply:
Unless you have a cmake target that is called "log"

Reply:
I think Abseil creates a target called log when built as part of the project (via FetchContent_Populate in this case).

See, e.g.
https://github.com/abseil/abseil-cpp/blob/master/absl/log/CMakeLists.txt#L231
where they also seem to have come to the same workaround.

Also, apologies, I believe the wasm build with onnxruntime-extensions enabled will still fail on this run. I have to do a bit of work because it has a lot of target_include_directories(... PUBLIC) for build dependencies that are prefixed in the build tree.

Reply:
Interesting to know that .